src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -345,4 +345,7 @@
   <data name="net_http_no_concurrent_io_allowed" xml:space="preserve">
     <value>The stream does not support concurrent I/O read or write operations.</value>
   </data>
+  <data name="net_http_unix_invalid_response" xml:space="preserve">
+    <value>The server returned an invalid or unrecognized response</value>
```

Comment:
Am leaving this for now, if you do not mind. Am using the error message in the validation and the exception message in the windows version does not contain the period.
If there is any change in the tests, I will add the period here.